SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs

```diff
@@ -331,5 +342,23 @@ private static string Quote(string toQuote)
         {
             return $"\"{toQuote}\"";
         }
+
+        private static void ForceSpatialNetworkingInConfig(string workerPath, string gameName)
+        {
+            var defaultGameIniPath = Path.Combine(workerPath, gameName, "Config", "DefaultGame.ini");
+
+            // Make sure we are adding the override to the correct file, and that the setting hasn't moved.
+            Trace.Assert(File.ReadAllText(defaultGameIniPath).Contains("bSpatialNetworking="));
```

Comment:
What's the default behavior if this flag isn't present in the .ini file? Could you test how the editor behaves if there's not explicit setting in there? It might just work in which case we probably don't want to fail if it's not there.

Reply:
Turns out that Unreal simply uses the default values if they aren't present in the `DefaultGame.ini`. If these settings aren't changed from the defaults, they are also never added to the `.ini` files. Checking whether the option is already listed in the `.ini` will therefore stop execution in the wrong cases. I have removed the check - thanks for noting this!